packages/react-core/src/components/Compass/CompassPanel.tsx

```diff
@@ -0,0 +1,55 @@
+import styles from '@patternfly/react-styles/css/components/Compass/compass';
+import { css } from '@patternfly/react-styles';
+
+interface CompassPanelProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content of the panel. */
+  children: React.ReactNode;
+  /** Additional classes added to the panel. */
+  className?: string;
+  /** Indicates the panel should have a pill border radius */
+  isPill?: boolean;
```

Comment:
Just thinking of it, but as part of the followup maybe we can go through any shared props amongst all the new compass stuff and make sure the verbiage is the same/similar.